etc/pyenv.d/install/ccache.bash

```diff
@@ -43,6 +43,10 @@ setup_ccache() {
   fi
 
   export CCACHE_BASEDIR="${PYTHON_BUILD_BUILD_PATH}"
+  # Use custom cache location to avoid interfering with user's regular usage
+  #
+  # This also allows the cache to be cleared and configured seperately
+  export CCACHE_DIR=$(pyenv root)/ccache
```

Comment:
How would Pyenv "interfere with user's regular usage"? IIRC CCache's database is basically a hash table of previous source files and compilation results.

If anything, consolidating it will increase the chance of cache hits!